plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessInFlightExit.sol

```diff
@@ -34,6 +34,11 @@ library PaymentProcessInFlightExit {
         uint16 inputIndex
     );
 
+    event InFlightBondReturnFailed(
```

Comment:
Why do we do this only for inflight bonds?

Reply:
We do it for standard exits. Check 'PaymentProcessStandardExit.sol'.